src/Symfony/Component/Security/Csrf/SameOriginCsrfTokenManager.php

```diff
@@ -227,9 +227,21 @@ public function onKernelResponse(ResponseEvent $event): void
      */
     private function isValidOrigin(Request $request): ?bool
     {
-        $source = $request->headers->get('Origin') ?? $request->headers->get('Referer') ?? 'null';
+        $target = $request->getSchemeAndHttpHost().'/';
+        $source = 'null';
```

Comment:
why using a string containing `null` here instead of an actual `null` value ?

Reply:
The `"null"` string is a possible value of the `Origin` header. This saves checking for one more case.
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin#description

Reply:
Hum, here you assume that we cannot receive `Origin: null` and `Referer: http://not-the-same-origin.com/`. From the doc it seems unlikely to expose the `Referer` with "privacy sensitive" origin.

Reply:
Yep, that's what we should do, isn't it?

Reply:
Yes;